libs/langgraph/langgraph/graph/state.py

```diff
@@ -415,6 +424,57 @@ def add_edge(self, start_key: Union[str, list[str]], end_key: str) -> Self:
         self.waiting_edges.add((tuple(start_key), end_key))
         return self
 
+    def add_sequence(
+        self,
+        nodes: Sequence[Union[RunnableLike, tuple[str, RunnableLike]]],
```

Comment:
```suggestion
        nodes: Sequence[Union[RunnableLike, tuple[str, RunnableLike]]],
        *,
```

Reply:
my personal preference for APIs -- is to always use a `*` before a named argument with a default. 

It forces devs to type more code when using the API but generally makes the resulting code easier and with less variations/